.github/workflows/devcontainer.yml

```diff
@@ -0,0 +1,17 @@
+name: 'devcontainer'
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3.5.2
+
+      - name: Build and run dev container task
+        uses: devcontainers/ci@v0.3.1900000328
+        with:
+          runCmd: yarn test
```

Comment:
i think you need to run `yarn install` here too

Reply:
yarn test is probably to slow as it would run the full test suite

Reply:
switched to `yarn build`